Chapter_12/README.md

```diff
@@ -1,2 +1,22 @@
-# Production-Ready-Applied-Deep-Learning
-Monitoring and maintenance at production level
\ No newline at end of file
+# Monitoring Deep Learning Endpoints in Production
+
+In this chapter, we discuss popular tools for monitoring DL models and alerting. 
+
+Common tools for monitoring are
+
+* [Prometheus](https://prometheus.io/)
+* [CloudWatch](https://aws.amazon.com/cloudwatch/)
+* [Grafana](https://grafana.com/)
+* [DataDog](https://www.datadoghq.com/)
+* [SageMaker Clarify](https://aws.amazon.com/sagemaker/clarify/)
+
+Popular tools for alerting include
+
+* [PagerDuty](https://www.pagerduty.com/)
+* [DynaTrace](https://www.dynatrace.com/)
+
+Out of the various tools we introduce, we will get our hands dirty with CloudWatch.
+
+# TODO: add notebooks
+* [Monitoring a SageMaker endpoint using CloudWatch](notebook)
+* [Monitoring an EKS endpoint using CloudWatch](notebook)
```

Comment:
@leninworld please add a functional notebook summarizing the operations we describe in the book.

Reply:
@ljj7975 We will only add SageMaker-related code and not EKS code. For EKS, it is implicitly logging and available in CloudWatch by default (no action needed further).